### DISCUSSION TITLE
FTXM-A Perfera 2024 - new Intelligent Eye != sensor mode

### DISCUSSION THREAD

**Discussion by ggaljoen** [revk/xESP32-Faikout#316] [Ideas]
My 2 new units FTXM20A2V1B  & FTXM35A2V1B  work great with the current Beta version. (added to wiki - suggestion sort in alphabetical order)

LED on the unit is controlled by G6 bit2&3: HIGH = 01xx / LOW = 10xx / off = 11xx

The new Intelligent Eye seems not to be the same as the sensor mode.
Only thing found, from IR-remote toggle, is response GG changes from 1 to F @ x1xx.
Nothing (visible?) changes when Intelligent Eye activation from the app.

Any command I can issue to test/activate?

**Comment by nikbyte** [comment]
The same issue on my FTXA35C2V1B Stylish. Any chance to fix it?

**Reply by sjaakie09** [reply]
Hello yes would be nice to have this sensor info also in home assistant 👍for rental solutions then we know if someone is in the house!can someone assist in this?

**Reply by revk** [reply]
Anything S21 can do, so if you find an S21 message for what you want. Let me know.
